### PR TITLE
Merge release/16.5 back into develop

### PR DESCRIPTION
**Pull request by loremattei**
Merge new release notes for 16.5 into `develop`.

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/metadata/PlayStoreStrings.po [hunk 1]
```diff
@@ -11,6 +11,18 @@ msgstr ""
...
+"16.5:\n"
+"Blocks: We added cross-post support for P2, full-width and wide alignment support in the Columns block, the ability to pick a link in the Image block, and the ability to move a block by long-pressing the up/down buttons.\n"
+"\n"
+"Activity Log: Track your site activity with filters that sort by date or type.\n"
+"\n"
+"Site Pages: Copy a page with the Duplicate button.\n"
+"\n"
+"\n"
+msgstr ""
+
```
Comment (AliSoftware) [inline WordPress/metadata/PlayStoreStrings.po:25]: Is it expected that we _don't_ have short release notes for 16.5 (i.e. a new entry for `release_note_short_165` key) as we did for e.g. 16.3?

Do we only add that entry if the full release notes looks too close to the 500 chars limit and Editorial provided an alternate shorter version and that's why we don't have any this time around?

File: WordPress/src/main/java/org/wordpress/android/ui/sitecreation/previews/SitePreviewViewModel.kt [hunk 2]
```diff
@@ -162,8 +162,11 @@ class SitePreviewViewModel @Inject constructor(
     private fun startCreateSiteService(previousState: SiteCreationServiceState? = null) {
         if (networkUtils.isNetworkAvailable()) {
             siteCreationState.apply {
+                // A non-null [segmentId] may invalidate the [siteDesign] selection
+                // https://github.com/wordpress-mobile/WordPress-Android/issues/13749
+                val segmentIdentifier = if (siteDesign != null) null else segmentId
                 val serviceData = SiteCreationServiceData(
-                        segmentId,
+                        segmentIdentifier,
```
Comment (AliSoftware) [inline WordPress/src/main/java/org/wordpress/android/ui/sitecreation/previews/SitePreviewViewModel.kt:169]: _For reference if another reviewer was wondering: this change comes from https://github.com/wordpress-mobile/WordPress-Android/pull/13751 (i.e. even if there was no new beta yet after this, this change is still expected)_

**Review by AliSoftware — APPROVED** [review]
LGTM, but will let you merge after you confirm about short release notes question

File: WordPress/metadata/PlayStoreStrings.po (hunk 1 above)
Comment (loremattei) [reply WordPress/metadata/PlayStoreStrings.po:25]: Yep, this is right!

**Comment by loremattei** [comment]
I'm closing this in favor of https://github.com/wordpress-mobile/WordPress-Android/pull/13765 which solves the merge conflict.

**Pull request closed without merging**